test/test_analysis.py

```diff
@@ -45,3 +50,54 @@ async def test_add_cram(self):
 
         self.assertEqual(1, analysis_samples[0]['sample_id'])
         self.assertEqual(analyses[0]['id'], analysis_samples[0]['analysis_id'])
+
+    @run_test_as_sync
+    async def test_get_sample_file_sizes(self):
```

Comment:
Could you add tests to cover the additional cases:

- Multiple CRAMs of the same sequencing type
- CRAMs from multiple samples